src/main/java/it/gov/pagopa/payment/connector/rest/reward/RewardCalculatorRestClient.java

```diff
@@ -17,14 +19,15 @@ public interface RewardCalculatorRestClient {
           produces = MediaType.APPLICATION_JSON_VALUE)
   @ResponseBody
   ResponseEntity<AuthPaymentResponseDTO> previewTransaction(@PathVariable("initiativeId") String initiativeId,
-                                                           @RequestBody AuthPaymentRequestDTO body);
+                                                           @RequestBody PreAuthPaymentRequestDTO body);
```

Comment:
I think we could use a more expressive name. We don't have a real pre-authorization phase, so we could refer to this DTO as a payment request, so `PaymentRequestDTO` I think is more appropriated